src/main/java/common/helpers/BrowserHelper.java

```diff
@@ -66,11 +63,11 @@ public static void maximizeBrowser() {
         getDriver().manage().window().maximize();
     }
 
-    public static void acceptAlert() {
+    public static void acceptAlert(){
```

Comment:
why?

Reply:
Because I think these 2 methods are easier and more straightforward to use. And most of all, my trainer Tan said that I should use this way.

Reply:
no, why remove the space? it's the correct space